### PR TITLE
Payment docs update attemp2

### PR DESCRIPTION
**Pull request by KBbitsP**
Added JS snippets for payment api

### CODE REVIEW

File: source/includes/_payment.md [hunk 1]
```diff
@@ -373,14 +418,23 @@ KillBillClient::Model::Payment.find_by_external_key(external_key,
...
-paymentApi = killbill.api.PaymentApi()
+paymentApi = killbill.PaymentApi()
+
 payment_external_key = 'sample_external_key' 
 
-paymentApi.get_payment_by_external_key(payment_external_key,
-                                       api_key,
-                                       api_secret)
+payment = paymentApi.get_payment_by_external_key(payment_external_key)
+```
```
Comment (reshmabidikar) [inline source/includes/_payment.md:426]: Is a new line required after the Python code?

**Review by reshmabidikar — COMMENTED** [review]
Shouldn't this PR be against the https://github.com/killbill/slate/tree/payment-docs-update branch?

File: source/includes/_payment.md (hunk 1 above)
Comment (KBbitsP) [reply source/includes/_payment.md:426]: Reshma, I think it is showing fine in local. 

image

File: source/includes/_payment.md [hunk 2]
```diff
@@ -1883,7 +2011,14 @@ paymentApi.modify_payment_custom_fields(payment_id,
                                         comment='comment')
 ```
 ```javascript
+const paymentApi: killbill.PaymentApi = new killbill.PaymentApi(config);
+
+const paymentId = 'f33e0adc-78df-438a-b920-aaacd7f8597a'
+const customFieldId = '9913e0f6-b5ef-498b-ac47-60e1626eba8f'
 
+const customFieldBody:killbill.CustomField[] = [{customFieldId:customFieldId,value:'test_modify_value'}];
```
Comment (reshmabidikar) [inline source/includes/_payment.md:2019]: consider renaming to `customFields`

File: source/includes/_payment.md [hunk 3]
```diff
@@ -1720,7 +1838,13 @@ paymentApi.create_payment_custom_fields(payment_id,
                                         comment='comment')
 ```
 ```javascript
+const paymentApi: killbill.PaymentApi = new killbill.PaymentApi(config);
+
+const paymentId = 'bfc12b3c-0042-48c5-b233-2d8e015f49f6';
+
+const customFieldBody: killbill.CustomField[] = [{ name: 'Test Custom Field', value: 'test_value' }];
```
Comment (reshmabidikar) [inline source/includes/_payment.md:1845]: consider renaming to `customFields`

File: source/includes/_payment.md [hunk 4]
```diff
@@ -884,7 +926,15 @@ paymentApi.chargeback_payment(payment_id,
                               comment='comment')
 ```
 ```javascript
+const paymentApi: killbill.PaymentApi = new killbill.PaymentApi(config);
+
+const transactionBody: killbill.PaymentTransaction = {amount: 100,
+                                                      paymentId: '536e3955-f69b-409f-8447-7cc24361f03f'
```
Comment (reshmabidikar) [inline source/includes/_payment.md:932]: code alignment looks a bit off here and in a few other places.

**Review by reshmabidikar — COMMENTED** [review]
Looks good, I've left some minor comments. Also, code alignment looks a bit off in some of the endpoints.

**Review by reshmabidikar — APPROVED** [review]
This PR contains too many changes so approving it for now. We can revisit it once it is merged.

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**